src/app/pages/research/people-section/people-section.js

```diff
@@ -0,0 +1,54 @@
+import React, {useState} from 'react';
+import TabGroup from '~/components/tab-group/tab-group.jsx';
+import ContentGroup from '~/components/content-group/content-group.jsx';
+import AccordionGroup from '~/components/accordion-group/accordion-group.js';
+import AlumniTab from './alumni-tab/alumni-tab';
+import MembersTab from './members-tab/members-tab';
+import './people-section.css';
+
+function TabAccordionCombo({children}) {
+    const labels = children.map((c) => c.props.label);
+    const initialSelection = children.findIndex((c) => c.props.selected) || 0;
+    const [selectedLabel, setSelectedLabel] = useState(labels[initialSelection]);
+    const selectedIndex = labels.indexOf(selectedLabel);
```

Comment:
will selectedLabel always be found, or do you need to deal with selectedIndex == -1 ?

Reply:
selectedLabel can only take on values from labels. It will always be found.